core/src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -186,6 +174,8 @@ public void electrocute(final LivingEntity lent) {
 		DamageHandler.damageEntity(lent, this.damage, this);
 		if (ThreadLocalRandom.current().nextDouble() <= this.stunChance) {
 			final MovementHandler mh = new MovementHandler(lent, this);
+			if (lent instanceof Player && BendingPlayer.getBendingPlayer((Player) lent).isAvatarState()) //Skip players in the AvatarState
+				return;
 			mh.stopWithDuration((long) this.stunDuration, Element.LIGHTNING.getColor() + "* Electrocuted *");
```

Comment:
Should Lightning, Paralyze, and Immobilize which use `MovementHandler::stopWithDuration` also have their ActionBar message be configurable?

Reply:
I agree they should be. But it was just out of the scope of what I did here and in my opinion, should be done in another update

Reply:
Once this gets merged, I'll make an issue for this 👍